controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -380,6 +380,11 @@ func (r *OCSInitializationReconciler) ensureOcsOperatorConfigExists(initialData
 			ocsOperatorConfigData[util.CsiRemoveHolderPodsKey] = ocsOperatorConfig.Data[util.CsiRemoveHolderPodsKey]
 		}
 
+		if r.allowRemoteStorageConsumers() {
+			ocsOperatorConfigData[util.CsiEnableCephFSKey] = "false"
+			ocsOperatorConfigData[util.CsiEnableRBDKey] = "false"
+		}
```

Comment:
The `allowRemoteStorageConsumers` indicator is for control over the API server and should be used only for that. These settings can and should be configured by the deploying agent via the storagecluster CR